ci/test/04_install.sh

```diff
@@ -33,7 +33,7 @@ export P_CI_DIR="$PWD"
 
 if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
   echo "Creating $DOCKER_NAME_TAG container to run in"
-  ${CI_RETRY_EXE} docker pull "$DOCKER_NAME_TAG"
+  docker build ${BASE_ROOT_DIR}/ci -t "$DOCKER_NAME_TAG" --build-arg DOCKER_NAME_TAG=$DOCKER_NAME_TAG
```

Comment:
I am not familiar with docker, does this overwrite the system image for, let's say, ubuntu:focal?

Reply:
ah, that's a good point, I should probably add something extra to the tag to make sure there are no weird conflicts later with your local images

Reply:
Maybe just keep everything as it was and only run `useradd ...` (and the other stuff from entry.sh after `docker run`?

Reply:
hm, I'd prefer to do it through the entrypoint script as it's more clear (to me, anyway) as to what's going on. It also keeps all the docker permissions stuff together, which feels better than mixing more logic for setting up docker into the install script